ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/ledger/ScenarioLoader.scala

```diff
@@ -190,17 +222,20 @@ object ScenarioLoader {
               case (nid, parties) => (nodeIdWithHash(nid), parties)
             }
             ledger +=
-              Transaction(
-                transactionId,
-                transactionId,
-                "scenario-loader",
-                richTransaction.committer,
-                workflowId,
-                time.toInstant,
-                time.toInstant,
-                recordTx,
-                recordDisclosure.transform((_, v) => v.toSet[String])
-              )
+              (
```

Comment:
why the extra `(`, `)`s?

Reply:
because otherwise it's invoking the method with two arguments

Reply:
oh, right